onnx/test/cpp/function_verify_test.cc

```diff
@@ -102,6 +103,175 @@ void VerifyTypeConstraint(const OpSchema& function_op, const FunctionProto* func
   ++counter;
 }
 
+// Testing the function-definitions provided for function-ops in ONNX schema registry.
```

Comment:
I was thinking to do similar things in the atenlib. I wonder if this can be exposed from onnx or adapted for onnxscript somehow?

Reply:
Something like that would be helpful, yes. But, for now, since the functions are available in ONNX's opschema, this was simpler, while in onnxscript we would want to do it for custom functions. We should add some abstraction to support both. I didn't want to round-trip the FunctionProto from C++ to Python and back to C++ for the tests below.